third_party/VENDOR-LICENSE/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
I think these are not needed. Can you update your go-licenses

```
go install github.com/google/go-licenses@master
```

Reply:
dropped